website/docs/developer/research/research-graphql.md

```diff
@@ -237,6 +286,18 @@ The following tools might be useful for developers while working with GraphQL
 - GraphQL Boilerplates- useful for backend projects.
 - Apollo Graph Manager-  useful for monitoring the performance and usage of GraphQL backend.
 
+### Recommendation
+While GraphQL is an interesting technology, the lack of documentation makes implementation difficult.
+In particular, deployment and integration lack vital instructions, with most tutorials running in a docker container or Python script rather than a realistic environment.
+Theoretically GraphQL could be run in an EC2 instance with some form of connecting layer such as [Fargate](https://aws.amazon.com/fargate/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc&fargate-blogs.sort-by=item.additionalFields.createdDate&fargate-blogs.sort-order=desc) but this is untested and would need research and cost analysis.
+
+Should this be desired, I feel strongly that Graphene is the only known option that gives us the flexibility that is required to justify this level of adaptation.
+Research would be required for:
+- Ideal deployment method
+- Accessing API
+- Security
+- GraphQL UI
```

Comment:
Since this would be used only for development, we could use something like [Insomnia](https://insomnia.rest/) to do this for us. We should not realistically deploy and GraphQL UI since that is not something we would want to expose in a production environment.

Reply:
My point is that we don't know if anything will work with Graphene. The only example I was able to get functional in AWS only accepts queries.

Reply:
I've added a note.